### ISSUE TITLE
SA1119 false positive when calling methods on `Range` literals

### ISSUE THREAD

**Issue by julealgon** [StyleCop/StyleCop#241]
The following code causes a SA1119:

```csharp
_ = (0..5).End;
```

The analyzer then suggests "removing the unnecessary parenthesis":

<img width="522" alt="rangeBug" src="https://user-images.githubusercontent.com/461579/71753405-4e626c80-2e61-11ea-8b4c-20d0766e9271.png">

Which then breaks the code:

<img width="577" alt="rangeBug2" src="https://user-images.githubusercontent.com/461579/71753475-7ce04780-2e61-11ea-93c2-f39882fa7800.png">
